src/isar_turtlebot/robotinterface.py

```diff
@@ -123,12 +134,39 @@ def log_status(
     def get_inspection_references(
         self, vendor_mission_id: Any, current_step: Step
     ) -> Sequence[Inspection]:
-        return []
+        now: datetime = datetime.utcnow()
+
+        if isinstance(current_step, TakeImage):
```

Comment:
There is supoort for scheduling a thermal image step but not anyway to get the inspection reference for thermal image. This should be supoorted for thermal image

Reply:
Will remove support for `ThermalImageStep` instead, and thermal image step scheduling can be handled in a separate PR.